section11/numberformat_diff.html

```diff
@@ -0,0 +1,1047 @@
+        1. If _f_ ≠ 0, then
+          1. Let _k_ be the number of characters in _m_.
+          1. If _k_ ≤ _f_, then
+            1. Let _z_ be the String consisting of _f_+1–_k_ occurrences of the character `"0"`.
+            1. Let _m_ be the concatenation of Strings _z_ and _m_.
+            1. Let _k_ be _f_+1.
+          1. Let _a_ be the first _k_–_f_ characters of _m_, and let _b_ be the remaining _f_ characters of _m_.
+          1. Let _m_ be the concatenation of the three Strings _a_, `"."`, and _b_.
+          1. Let _int_ be the number of characters in _a_.
+        1. Else, let _int_ be the number of characters in _m_.
+        1. Let _cut_ be _maxFraction_ – _minFraction_.
+        1. Repeat, while _cut_ > 0 and the last character of _m_ is `"0"`
+          1. Remove the last character from _m_.
+          1. Decrease _cut_ by 1.
+        1. If the last character of _m_ is `"."`, then
+          1. Remove the last character from _m_.
+        1. <del class="block">If _int_ < _minInteger_, then</del>
+          1. <del class="block">Let _z_ be the String consisting of _minInteger_–_int_ occurrences of the character `"0"`.</del>
+          1. <del class="block">Let _m_ be the concatenation of Strings _z_ and _m_.</del>
+        1. <del class="block">Return _m_.</del>
+        1. <ins class="block">Return the Record { [[FormattedString]]: _m_, [[RoundedNumber]]: _xFinal_, [[IntegerDigitsCount]]: _int_ }.</ins>
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-unwrapnumberformat" aoid="UnwrapNumberFormat">
+      <h1>UnwrapNumberFormat( _nf_ )</h1>
+      <p>
+        The UnwrapNumberFormat abstract operation gets the underlying NumberFormat operation
+        for various methods which implement ECMA-402 v1 semantics for supporting initializing
+        existing Intl objects.
+      </p>
+      <emu-alg>
+        1. Assert: Type(_nf_) is Object.
+      </emu-alg>
+      <emu-normative-optional>
+      <emu-alg>
+        2. If _nf_ does not have an [[InitializedNumberFormat]] internal slot and ? InstanceofOperator(_nf_, %NumberFormat%) is *true*, then
+          1. Let _nf_ be ? Get(_nf_, %Intl%.[[FallbackSymbol]]).
+      </emu-alg>
+      </emu-normative-optional>
+      <emu-alg>
+        3. If Type(_nf_) is not Object or _nf_ does not have an [[InitializedNumberFormat]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return _nf_.
+      </emu-alg>
+      <emu-note>See <emu-xref href="#legacy-constructor"></emu-xref> for the motivation of the normative optional text.</emu-note>
+    </emu-clause>
+
+    <ins class="block">
+    <emu-clause id="sec-setnumberformatunitoptions" aoid="SetNumberFormatUnitOptions">
+      <h1>SetNumberFormatUnitOptions ( _intlObj_, _options_ )</h1>
+      <p>
+        The abstract operation SetNumberFormatUnitOptions resolves the user-specified options in the units realm onto the unit options object.
```

Comment:
The word "realm" is linking to the definition of JS Realm. Is that intentional?

Reply:
Fixed.